pkg/controller/migmigration/backup.go

```diff
@@ -199,7 +269,12 @@ func (t *Task) getReplicatedBackup() (*velero.Backup, error) {
 	if err != nil {
 		return nil, err
 	}
-	if len(list.Items) > 0 {
+	// If this is stage we only need to wait for 1 backup to be replicated
+	if len(list.Items) == 1 && t.stage() {
+		return &list.Items[0], nil
+	}
+	// If this is not a stage, we need to find 2 backups before continuing
+	if len(list.Items) > 1 {
 		return &list.Items[0], nil
```

Comment:
Is it intentional that when the list contains multiple items, we only return the first item in the list here?

Reply:
It's intentional but I'm not sure if it's really the clearest way to do this... basically I just wanted to return *something* to tell the task runner that the backups were both replicated. The task runner doesn't do anything with it AFAIK. I think that I will change this function to simply return a `bool` to tell us if the backup or backups were properly replicated